Core/DISET/private/Transports/SSL/SocketInfo.py

```diff
@@ -169,21 +169,16 @@ def __getCAStore( self ):
               if crl.has_expired():
                 continue
               crlID = crl.get_issuer().one_line()
-              crlNotAfter = crl.get_not_after()
-              if crlID not in crlsDict:
-                crlsDict[ crlID ] = ( crlNotAfter, crl )
-                crlsFound += 1
-              else:
-                if crlsDict[ crlID ][0] < crlNotAfter:
-                  crlsDict[ crlID ] = ( crlNotAfter, crl )
+              crlsDict[ crlID ] = crl 
+              crlsFound += 1
               continue
-            except:
+            except Exception, e:
```

Comment:
Just to be super picky, the proper way of doing that is 
except Exception as e (pep8 recommendation, and compatible with python 3)
But well done for spotting the bug :-)